openapi/components/schemas/Credentials/Credential.yaml

```
@@ -9,7 +9,12 @@ properties:
     type: string
     enum:
       - active
+      - inactive
       - deactivated
+    x-enumDescriptions:
+      active: Credentials active and can be used.
+      inactive: Credentials temprorary inactivated and cannot be used, but can be activated again.
```

nitpick: I'm not sure if "inactive" is the best term to use in the description text (it's fine as an enum)

"inactive" does not convey that the credentials cannot be used. I think temporarily deactivated is more accurate because the credentials cannot be used while in this state.

Also, (if possible -- I'm not sure how relevant this is) I think it would be good to describe how to activate inactive credentials here, or point to a location where this is described

```suggestion
      inactive: Credentials are temporarily deactivated and cannot be used until reactivated.
```

---

Updated based on our conversation 
```suggestion
      inactive: Credentials are temporarily inactivated and cannot be used and cannot be used until reactivated.
```